### PR TITLE
More models

### PR DESCRIPTION
**Pull request by xiaolong0728**
First version of adapting to arg_parse. Small updates on the evaluation metric (adding mean, median, etc.)

### CODE REVIEW

File: common_utils/generate_metric_dict.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# There are negative values, so errors occur when calculating MSLE
+# Brier score is used for binary and categorical outcomes that can be structured as true or false
+# There are no classes in data, so we cannot calculate roc_auc_score, ap_score
+import properscoring as ps
+from sklearn.metrics import mean_squared_error, mean_absolute_error, mean_squared_log_error, brier_score_loss, average_precision_score, roc_auc_score
+from utils_evaluation_metrics import EvaluationMetrics
+from views_forecasts.extensions import *
+
+def generate_metric_dict(df, steps, depvar):
+    evaluation_dict = EvaluationMetrics.make_evaluation_dict(steps=steps[-1])
```
Comment (Polichinel) [inline common_utils/generate_metric_dict.py:11]: I need a docstring here. 
I'm unsure which df it takes, but I suspect one with predictions and true etc.
You could of course know this last week, but we need to align that with the new data class for predictions. Bu we can do that later as long as it goes firmly on a tode list.

Additionally, if this is to be general it should take all to metrics defined in the workshop. But of course only fill in the ones relevant for the specific model. 

I have similar code in the PR for aligning eval and abstracting data loaders so we can try to align later.

Point: No need to do stuff now, but this should go on the todo list

File: models/lavender_haze/artifacts/model_calib_partition.pkl
Comment (Polichinel) [inline models/lavender_haze/artifacts/model_calib_partition.pkl:1]: We should save pkl here but I also think I have added pkl to gitignore now. So these just have to come down and then we're good

**Review by Polichinel — COMMENTED** [review]
Super - If you put the thing about the first function in a todo in common_utils/README.md
And get the pkl files down at some point
Then just merge to main
